docs/reference/result.rst

```diff
@@ -30,27 +30,13 @@ to combine results with asynchronouse code.
 .. code-block:: typescript
 
     // The actual signature is more complicated but this should be good enough.
-    static all(...results: Result<T, E>): Result<T[], E>
     static all(results: Result<T, E>[]): Result<T[], E>
 
 Parse a set of ``Result``, returning an array of all ``Ok`` values.
 Short circuits with the first ``Err`` found, if any.
 
 Example:
 
-.. code-block:: typescript
-
-    let pizzaResult: Result<Pizza, GetPizzaError> = getPizzaSomehow();
-    let toppingsResult: Result<Toppings, GetToppingsError> = getToppingsSomehow();
-
-    let result = Result.all(pizzaResult, toppingsResult); // Result<[Pizza, Toppings], GetPizzaError | GetToppingsError>
-
-    let [pizza, toppings] = result.unwrap(); // pizza is a Pizza, toppings is a Toppings.  Could throw GetPizzaError or GetToppingsError.
-
-When working with a set of results of unknown size, you can use the array version of ``all()``.
-
-Example:
-
```

Comment:
The only downside of this PR is that our pizza/topping example is now gone 😄

Reply:
I got you! We had duplicated examples (parameter spread vs array) and the examples are still there for the array versions.